hack/dump_scenario_from_k8s.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+set -e
+
+readonly ARGS=("$@")
+
+usage() {
+    cat <<-EOF
+    usage: dump_scenario_from_k8s.sh options
+
+    Dump the current scenario from a running trento-server installation on the k8s cluster
+
+    OPTIONS:
+        -n, --name            The name to use for the scenario. Defaults to "current".
+        -p, --path            The path where the scenario should be saved. Defaults to the current directory.
+        -h, --help            Print this help.
+
+    Example:
+        dump_scenario_from_k8s.sh --name failover --path /tmp
+EOF
+}
+
+cmdline() {
+    local arg=
+
+    for arg; do
+        local delim=""
+        case "$arg" in
+        --name) args="${args}-n " ;;
+        --path) args="${args}-p " ;;
+        --help) args="${args}-h " ;;
+
+        # pass through anything else
+        *)
+            [[ "${arg:0:1}" == "-" ]] || delim="\""
+            args="${args}${delim}${arg}${delim} "
+            ;;
+        esac
+    done
+
+    eval set -- "$args"
+
+    while getopts "n:p:h" OPTION; do
+        case $OPTION in
+        h)
+            usage
+            exit 0
+            ;;
+        n)
+            readonly NAME=$OPTARG
+            ;;
+        p)
+            readonly EXPORT_PATH=$OPTARG
+            ;;
+        *)
+            usage
+            exit 0
+            ;;
+        esac
+    done
+
+    return 0
+}
+
+dump-scenario() {
+    local name=${NAME:-current}
+    local path="${EXPORT_PATH:-$PWD}"
+
+    if [[ -d "$path/scenarios/$name" ]]; then
+        echo "The scenario $name already exists in $path/scenarios/$name"
+        echo "Please choose a different name."
+        exit 1
+    fi
+
+    kubectl exec -ti deploy/trento-server-web -- /app/trento ctl dump-scenario --name="$name" --path /scenarios
```

Comment:
Hey @fabriziosestito ,
I'm facing some issues running this commands. (using them to debug a current bug):
```
2022/01/14 14:58:29 github.com/trento-project/trento/internal/db/db.go:29
[error] failed to initialize database, got error failed to connect to `host=localhost user=postgres database=trento`: dial error (dial tcp [::1]:5432: connect: connection refused)
FATA[0000] Error while initializing the database: failed to connect to `host=localhost user=postgres database=trento`: dial error (dial tcp [::1]:5432: connect: connection refused) 
command terminated with exit code 1

```
And if I put the `--db-host` using the postrgres address:

```
FATA[0000] Error while writing event: open /scenarios/db_nil/f98d4781-77f0-5392-b8e8-fe5b8e8263d6/sap_system_discovery: no such file or directory 
command terminated with exit code 1

```

Reply:
both fixed in the current PR as of https://github.com/trento-project/trento/pull/708/commits/828fba713d12cf7c2f15c67c0c7ccbbc795ce39f and https://github.com/trento-project/trento/pull/708/commits/b1983d3c21fc92889d1a7abe9fa4d4e771022be0